apps/src/redux/instructions.js

```diff
@@ -4,73 +4,63 @@
  * that are required for this feature, and the reducer that sets state based
  * off of those actions.
  */
-var _ = require('../lodash');
+const TOGGLE_INSTRUCTIONS_COLLAPSED = 'instructions/TOGGLE_INSTRUCTIONS_COLLAPSED';
+const SET_INSTRUCTIONS_HEIGHT = 'instructions/SET_INSTRUCTIONS_HEIGHT';
+const SET_INSTRUCTIONS_MAX_HEIGHT = 'instructions/SET_INSTRUCTIONS_MAX_HEIGHT';
 
-var TOGGLE_INSTRUCTIONS_COLLAPSED = 'instructions/TOGGLE_INSTRUCTIONS_COLLAPSED';
-var SET_INSTRUCTIONS_HEIGHT = 'instructions/SET_INSTRUCTIONS_HEIGHT';
-var SET_INSTRUCTIONS_MAX_HEIGHT = 'instructions/SET_INSTRUCTIONS_MAX_HEIGHT';
-
-var instructionsInitialState = {
+const instructionsInitialState = {
   collapsed: false,
   // represents the uncollapsed height
   height: 300,
   maxHeight: 0
 };
 
-module.exports.default = function reducer(state, action) {
-  state = state || instructionsInitialState;
-
+export default function reducer(state = instructionsInitialState, action) {
   if (action.type === TOGGLE_INSTRUCTIONS_COLLAPSED) {
-    return _.assign({}, state, {
+    return Object.assign({}, state, {
       collapsed: !state.collapsed
     });
   }
 
   if (action.type === SET_INSTRUCTIONS_HEIGHT &&
       action.height !== state.height) {
-    return _.assign({}, state, {
+    return Object.assign({}, state, {
       height: action.height
     });
   }
 
   if (action.type === SET_INSTRUCTIONS_MAX_HEIGHT &&
       action.maxHeight !== state.maxHeight) {
-    return _.assign({}, state, {
+    return Object.assign({}, state, {
       maxHeight: action.maxHeight
     });
   }
 
   return state;
-};
+}
 
 /**
  * Toggles whether instructions are currently collapsed.
  */
-module.exports.toggleInstructionsCollapsed = function () {
-  return {
-    type: TOGGLE_INSTRUCTIONS_COLLAPSED
-  };
-};
+export const toggleInstructionsCollapsed = () => ({
+  type: TOGGLE_INSTRUCTIONS_COLLAPSED
+});
 
 /**
  * Set the height of the instructions panel
  * @param {number} height - Height of instructions pane
  */
-module.exports.setInstructionsHeight = function (height) {
-  return {
-    type: SET_INSTRUCTIONS_HEIGHT,
-    height: height
-  };
-};
+export const setInstructionsHeight = height => ({
```

Comment:
We're going to have to discuss style again, but I see that this is correct per [airbnb JS rule 8.4](https://github.com/airbnb/javascript#arrows--one-arg-parens).